src/EditorFeatures/CSharpTest/Formatting/Indentation/SmartTokenFormatterFormatRangeTests.cs

```diff
@@ -2855,21 +2855,22 @@ public async Task NoLineChangeWithSyntaxError()
 class Program{
     void Main()
     {
-        var f = new Goo { member;$$ }
+        var f = new Goo { member,$$ }
     }
 }";
 
             var expected = @"struct Goo { public int member; }
 class Program{
     void Main()
     {
-        var f = new Goo { member; }
+        var f = new Goo { member, }
     }
 }";
 
-            await AutoFormatOnSemicolonAsync(
+            await AutoFormatOnMarkerAsync(
                 code,
                 expected,
+                SyntaxKind.CommaToken,
                 SyntaxKind.OpenBraceToken);
         }
```

Comment:
please don't change existing test start-contents.  Just show what teh new behavior is.  Add new tests with the new start-contents.